### PR TITLE
NO_COLOR environment variable support

### LINKED ISSUE #788: Add NO_COLOR support
Check the NO_COLOR environment variable to turn off color support.
See https://no-color.org.

### PR DESCRIPTION
**Pull request by paper-lark**
Fixes #788

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -19,3 +19,6 @@ parts
 prime
 stage
 *.snap
+
+# IntelliJ IDEA files
+.idea
```
Comment (ariasuni) [inline .gitignore:24]: I think you should put this kind of stuff in your global gitignore.

File: man/exa.1.md [hunk 2]
```diff
@@ -224,6 +224,12 @@ Specifies the number of spaces to print between an icon (see the ‘`--icons`’
 
 Different terminals display icons differently, as they usually take up more than one character width on screen, so there’s no “standard” number of spaces that exa can use to separate an icon from text. One space may place the icon too close to the text, and two spaces may place it too far away. So the choice is left up to the user to configure depending on their terminal emulator.
 
+## `NO_COLOR`
+
+Disables ANSI colour in the output (regardless of its value). Can be overridden by `--color` option.
```
Comment (ariasuni) [inline man/exa.1.md:229]: I’m not sure that specifying «ANSI» is useful. Also, I think adding a link to the website would be nice.

```suggestion
Disables colours in the output (regardless of its value). Can be overridden by `--color` option. See https://no-color.org/.
```

**Review by ariasuni — CHANGES_REQUESTED** [review]
Nice, thanks! I’d like a very few things changes before I merge this, please look at my comments!

File: man/exa.1.md (hunk 2 above)
Comment (paper-lark) [reply man/exa.1.md]: Thanks, I removed ANSI word. As for the link it was already included on the next line.

**Comment by paper-lark** [comment]
@ariasuni Thank you for the feedback! I've fixed everything based on your comments

**Comment by ariasuni** [comment]
Thanks for you contribution :slightly_smiling_face:

**Comment by ariasuni** [comment]
Would you like @paper-lark to contribute to the no-color website and add exa there as well?

**Comment by paper-lark** [comment]
> Would you like @paper-lark to contribute to the no-color website and add exa there as well?

Sure. Though maybe we should release a new version of exa with NO_COLOR support first

**Comment by ariasuni** [comment]
I’m not in charge of making new releases, and it can take a while, so I figured out it would be nice to add it. And when a new release of exa is published, it’s possible that we forget to update the no-color website, so…

**Comment by paper-lark** [comment]
Oh, sure, I'll update the website then

**Pull request merged**